src/dom/dom.js

```diff
@@ -3283,14 +3285,14 @@ p5.Element.prototype.size = function (w, h) {
         this.elt.height = aH;
       }
 
-      this.width = this.elt.offsetWidth;
-      this.height = this.elt.offsetHeight;
+      originalWidth = aW;
+      originalHeight = aH;
 
       if (this._pInst && this._pInst._curElement) {
         // main canvas associated with p5 instance
         if (this._pInst._curElement.elt === this.elt) {
-          this._pInst._setProperty('width', this.elt.offsetWidth);
-          this._pInst._setProperty('height', this.elt.offsetHeight);
+          this._pInst._setProperty('width', originalWidth);
```

Comment:
Is there a reason for having `originalWidth/Height` as separate variables, or can we simplify this a bit and just use `aW` and `aH` here?

Reply:
Ah yes! We can just use aW and aH here. I don't think we need to have separate local variables `orignalWidth/Height`.

Reply:
Ah yes! I realised it later that `aW` and `aH` can be used instead of `originalWidth/Height`.